smartsim/_core/_install/builder.py

```diff
@@ -174,7 +182,7 @@ def run_command(self,
             if proc.returncode != 0:
                 raise BuildError(error)
         except (OSError, SubprocessError) as e:
-            raise BuildError(e)
+            raise BuildError(e) from e
```

Comment:
This change seems redundant

Reply:
the `from` ensures that the exception information is included with the newly raised exception. Since there is no logging here, the from should be critical to identify where the original exception occurred when handling the BuildError